packages/automerge-repo/test/helpers/DummyStorageAdapter.ts

```diff
@@ -0,0 +1,19 @@
+import { DocumentId, StorageAdapter } from "../../src"
```

Comment:
great! now we can remove the memory adapter (remember we should also remove the NPM module)